Hiring managers lessons learned

davidfischer opened the pull request:
The goal of this blog is content marketing. I do think, however, that we got a ton of feedback from hiring managers that is useful to other hiring managers so getting the word out is a good idea.

## Thoughts/Questions

- I tried to keep the blog somewhat focused on lessons for hiring managers. I will do a separate post on lessons for candidates. I also wanted to keep the post from being too much of a brain dump.
- Obviously I didn't name names or companies. We didn't get permission to share any of that. It would make for a more compelling post but I think it's ok without it.
- I didn't touch on diversity and inclusivity which we heard unprompted in about a half dozen interviews. Thoughts on sharing that?

davidfischer commented on lessons-from-hiring-manager-interviews.rst:
```diff
@@ -0,0 +1,165 @@
+Reaching people who aren't looking is critical
+----------------------------------------------
+
+One of our biggest takeaways at Read the Docs was that companies wanted to reach candidates before they started actively looking.
```
```suggestion
One of our biggest takeaways from the interviews was that companies wanted to reach candidates before they started actively looking.
```

davidfischer commented:
> I didn't touch on diversity and inclusivity which we heard unprompted in about a half dozen interviews. Thoughts on sharing that?

I'm thinking something like this:


```rst
Diversity and inclusivity in hiring
-----------------------------------

We didn't ask any questions around hiring for diversity
but fully six hiring managers volunteered information on their companies' strategies
for getting more diverse candidates.
This is why I love this kind of interview where the goal is just to listen to what hiring managers say
and dig deeper when you hear something interesting.
Some of the larger companies we talked to had explicit diversity goals set at the company level.
Even among the hiring managers at smaller companies,
they shared some tactics they used to get more diverse candidates to apply.

Of the companies with hard diversity goals, most of them told us that their strategy
entailed syndicating their job openings to diversity focused jobs sites
or in some cases engaging directly with relevant interest groups.
In terms of efficacy, reviews of these various jobs boards was mixed.
One hiring manager said that posting to some of these sites
seemed like "checking a box" rather than a good way to actually get diverse applicants.

On the more positive side, hiring managers cited 
Girl Develop It, Black Girls Code, and PyLadies as fantastic channels
for **both increasing candidate diversity and for high quality applicants**.
Two companies also talked about trying to decrease bias
in their job postings using services like Textio.

Having multiple people mention diversity completely unprompted really shows that this topic
is becoming higher priority for companies and hiring managers.
It also shows that it if hiring for diversity is important to your company,
you need to have a plan for actually making those goals a reality.
It's worth checking whether the strategy actually works too.
```

ericholscher commented on lessons-from-hiring-manager-interviews.rst:
```diff
@@ -0,0 +1,165 @@
+Lessons From and For Hiring Managers
+====================================
+
+Over the last four weeks, Eric Holscher and I did dozens of customer development interviews with engineering hiring managers.
```
```suggestion
Over the last four weeks, the Read the Docs team did dozens of customer development interviews with engineering hiring managers.
```

ericholscher commented on lessons-from-hiring-manager-interviews.rst:
```diff
@@ -0,0 +1,165 @@
+We talked to people looking for talent at five person companies all the way up to the biggest names in tech.
+In this post, I am going to cover some of the common things we heard from hiring managers
+and **share some ways for hiring managers to improve their company's process**.
+In a week or two, I will write the second part to this blog with actionable tips
```
I've generally learned it's good to not make specific time promises. 

```suggestion
In our next post in this series, I will write the second part to this blog with actionable tips
```

ericholscher commented on lessons-from-hiring-manager-interviews.rst:
```diff
@@ -0,0 +1,165 @@
+The second best channel was a bit of a surprise with local meetups and local communities being very high in multiple interviews
+although this was less true among the largest companies we talked to.
+Companies also had some success with recruiting at conferences, using various job sites, and proactive outreach.
+I'll address recruiters in a separate section.
```
Not sure this line is needed.

ericholscher commented on lessons-from-hiring-manager-interviews.rst:
```diff
@@ -0,0 +1,165 @@
+In each interview, we asked whether the company used recruiters or recruiting related services (eg. Hired, Triplebyte).
+Universally, hiring managers brought up how expensive recruiters are,
+but most of the people we talked to who had actually used recruiters in the past had a positive impression overall.
+Hiring takes a ton of time and typically takes it from senior folks at the company.
```
```suggestion
Hiring takes a ton of time, typically from senior folks at the company.
```

ericholscher commented on lessons-from-hiring-manager-interviews.rst:
```diff
@@ -0,0 +1,165 @@
+and a hiring manager who posted to a very popular remote-only job board told me he got "literally one thousand applications".
+While I believe this is a legitimate complaint,
+I think `Aline Lerner said it best`_ when she said "Engineering hiring isn't a filtering problem. It's a sourcing problem."
+Filtering is just a time-consuming task that hiring managers directly deal with.
```
This feels a bit dismissive and negative, especially the heading. Perhaps "Sourcing people is the real problem" is a better angle.

ericholscher commented on lessons-from-hiring-manager-interviews.rst:
```diff
@@ -0,0 +1,165 @@
+What hiring managers are looking for is qualified applicants, which is harder to gauge without some additional work.
+One of the problems here is that a lot of jobs sites make it really easy for candidates to apply to lots of jobs.
+This is understandable from their perspective as more applicants makes it look like their platform is more valuable.
+I'm not trying to talk trash on these platforms as there's no reason for them to erect artificial barriers to the application funnel,
```
This feels a bit weird and defensive. Probably a better way to say this.

ericholscher commented on lessons-from-hiring-manager-interviews.rst:
```diff
@@ -0,0 +1,165 @@
+Instead of building product here, I think hiring managers just need a few tweaks
+to get more of a signal of candidate quality and intent in their applications.
+A few of the hiring managers we talked to described how they did this.
+Multiple hiring managers mentioned how they required a cover letter that answered specific questions ("why us" was the most popular).
```
Might be good to put these in a list to make it easier to skim. Otherwise it will likely get caught on a skim.

ericholscher commented on lessons-from-hiring-manager-interviews.rst:
```diff
@@ -0,0 +1,165 @@
+.. _Aline Lerner said it best: http://blog.alinelerner.com/building-a-product-in-the-technical-recruiting-space-read-this-first/
+
+
+TLDR
```
I've a bit less excited about TLDR -- perhaps "In summary"

ericholscher commented on lessons-from-hiring-manager-interviews.rst:
```diff
@@ -0,0 +1,165 @@
+  Great developers (especially senior ones) probably aren't actively looking for a new job and it's a job-seekers market.
+  Even if somebody isn't considering a new job at all, the next time they do, you want them to think of your company.
+* **Get more signal in your application process** by adding a step such as a cover letter, quiz,
+  or simply add a couple questions folks have to answer.
```
I wonder if we should put this at the top (where a tl;dr would normally go) as an intro, in order to get people to read further. I feel like this post is probably too long for most people to read, so having more of a hook would be good.

ericholscher commented:
+1 to adding the D&I content. I think it would be remiss of us to not mention it when talking about hiring. The "checking a box" line feels a bit tricky, might be a better way to phrase that where it isn't interpreted as against D&I hiring.

davidfischer commented:
> The "checking a box" line feels a bit tricky, might be a better way to phrase that where it isn't interpreted as against D&I hiring.

Very fair. I actually reworded it a couple times to sound less negative. It actually wasn't that negative in reality as that person was all for D&I but didn't think that those particular sites actually resulted in candidates. They had other ideas.

davidfischer commented:
I added:

- A summary at the top
- A diversity and inclusion section
- A thanks section at the bottom

I also fixed up a few of the more awkward phrases.

It's still a big block of text but hopefully people at least read the summary to catch their interest.
